src/main/java/frc/robot/WristSubsystem.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicExpoVoltage;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.EnumMap;
+import java.util.Map;
+
+public class WristSubsystem extends SubsystemBase {
+    private final TalonFX wristMotor;
+    private final CANcoder wristEncoder;
+    public static final InvertedValue motorInversion = InvertedValue.Clockwise_Positive;
+    public static final NeutralModeValue neutralMode = NeutralModeValue.Brake;
+
+    public enum Positions {
+        VERTICAL,
+        HORIZONTAL
+    }
+
+    private final EnumMap<Positions, Integer> positionsEnumMap =
+            new EnumMap<>(
+                    Map.ofEntries(
+                            Map.entry(Positions.VERTICAL, 90), Map.entry(Positions.HORIZONTAL, 0)));
```

Comment:
Formatting:

```suggestion
    private final EnumMap<Positions, Integer> positionsEnumMap = new EnumMap<>(
        Map.ofEntries(
            Map.entry(Positions.VERTICAL, 90),
            Map.entry(Positions.HORIZONTAL, 0)
        )
    );
```

Makes it easier to read individual entries.

Reply:
Would also recommend ordering smallest --> greatest